beacon_node/lighthouse_network/src/peer_manager/mod.rs

```diff
@@ -330,27 +326,25 @@ impl<TSpec: EthSpec> PeerManager<TSpec> {
             //    considered a priority. We have pre-allocated some extra priority slots for these
             //    peers as specified by PRIORITY_PEER_EXCESS. Therefore we dial these peers, even
             //    if we are already at our max_peer limit.
-            if (min_ttl.is_some()
-                && connected_or_dialing + to_dial_peers.len() < self.max_priority_peers()
-                || connected_or_dialing + to_dial_peers.len() < self.max_peers())
-                && self.network_globals.peers.read().should_dial(&peer_id)
```

Comment:
`should_dial` is done now inside `dial_peer` as it was being called every time `dial_peer` was also being called, see 191e0ae9c7a91ee5c89d4e774fcee542d8458789

Reply:
Yeah, nice